langchain4j-google-ai-gemini/src/main/java/dev/langchain4j/model/googleai/GeminiService.java

```diff
@@ -1,85 +1,94 @@
 package dev.langchain4j.model.googleai;
 
-//import io.reactivex.rxjava3.core.Observable;
-import okhttp3.OkHttpClient;
-import okhttp3.logging.HttpLoggingInterceptor;
+import com.google.gson.Gson;
```

Comment:
There is a ticket to migrate from Gson to Jackson. Let's not pull the Gson in

Reply:
Up until now, only Gson has been used in this module, but sure, I can switch to Jackson as well.

Reply:
When using Jackson, each (sub)class will require a dedicated no-argument constructor due to the use of the @Builder annotation within the POJOs. Should I continue, or would it be better to create a separate issue for that?

Reply:
@Bjarne-Kinkel let's keep it separate then